test/lit/passes/j2cl-inline.wast

```diff
@@ -2,7 +2,7 @@
 
 ;; NOTE: In real world example  no-inline would use _<once>_ but there is escaping problem in a multi-platform
```

Comment:
If escaping is fixed, this comment is now stale.

Reply:
Updated j2cl-inline.wast to show that without the inlining it has the same result as before.